packages/plugins/rubygem-foreman_leapp/rubygem-foreman_leapp.spec

```diff
@@ -0,0 +1,123 @@
+# template: foreman_plugin
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+%{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
+
+%global gem_name foreman_leapp
+%global plugin_name leapp
+%global foreman_min_version 1.25.0
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.0.1
+Release: 1%{?foremandist}%{?dist}
+Summary: A Foreman plugin for Leapp utility
+Group: Applications/Systems
+License: GPLv3
+URL: https://github.com/oamg/foreman_leapp
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+# start specfile generated dependencies
+Requires: foreman >= %{foreman_min_version}
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
+Requires: %{?scl_prefix}rubygem(foreman_remote_execution) >= 3.1.0
+Requires: %{?scl_prefix}rubygem(foreman_remote_execution) < 3.2
```

Comment:
this may be too strict, REX don't change that dramatically, I'd personally remove this line entirely but that means we should update the gemspec too and set the rex dependency as >= 3.1 at https://github.com/oamg/foreman_leapp/blob/master/foreman_leapp.gemspec#L20

Reply:
Created PR https://github.com/oamg/foreman_leapp/pull/48